app/Http/Controllers/Company/Employee/EmployeeSkillController.php

```diff
@@ -26,7 +27,7 @@ public function search(Request $request): JsonResponse
         $loggedCompany = InstanceHelper::getLoggedCompany();
 
         $name = $request->input('searchTerm');
-        $name = StringHelper::removeLettersWithAccent($name);
+        $name = Str::of($name)->ascii();
```

Comment:
Are you sure of this change? Is it really a replacement?

Reply:
It has to be tested ;)